pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -169,7 +237,10 @@ def use_dp(self):
 
     @property
     def use_ddp(self):
-        return self._distrib_type in (DistributedType.DDP, DistributedType.DDP_SPAWN)
+        return self._distrib_type in (
+            DistributedType.DDP, DistributedType.DDP_SPAWN, DistributedType.DDP_SHARDED,
+            DistributedType.DDP_SHARDED_SPAWN
+        )
```

Comment:
I'm not sure how I feel about this, as I think this means for my custom distributed plugin I'll need to modify the connector to get it working right? Is there any way these Enums could live as properties within the training type plugin so we can do:

```python
@property
def use_ddp(self):
    return self._training_type_plugin.is_distributed_training if self._training_type_plugin else False
```

Reply:
I think actually we should do `isinstance(plugin, DDPPlugin)` so that users can subclass it as much as they want
same also for the accelerator stuff
problem is, there are many tests with this _distrib_type and they will fail, as this is currently the logic also in dev 1.2

Reply:
ah yeah solid idea, this is something that can be addressed after i think!